src/index.css

```diff
@@ -14,10 +14,18 @@ body{
   height: 100%;
   width: 100%;
 }
+
+.headingfont {
+  font-family: "Love Ya Like A Sister", cursive;
+  font-weight: 400;
+  font-style: normal;
+}
+
```

Comment:
Instead of doing this, you can try add Arbitrary tailwind utility class.

e.g: `font-['Love_Ya_Like_A_Sister',cursive]`

```html
<h1 className="font-['Love_Ya_Like_A_Sister',cursive] text-4xl font-bold text-slate-100 p-2">
   Draw it Out!
</h1>
```

Reply:
Performed the changes as said.

Reply:
Please check & update if the PR can be merged.

Reply:
@0xabdulkhalid I hope the changes are alright & please update if the PR can be merged.

Reply:
Good 👍🏻